consensus/safety-rules/src/safety_rules_manager.rs

```diff
@@ -80,7 +80,9 @@ pub struct SafetyRulesManager {
 impl SafetyRulesManager {
     pub fn new(config: &mut SafetyRulesConfig) -> Self {
         match &config.service {
-            SafetyRulesService::Process(conf) => return Self::new_process(conf.server_address()),
+            SafetyRulesService::Process(conf) => {
+                return Self::new_process(conf.server_address(), config.timeout)
+            }
```

Comment:
we could potentially just pass in the config if it were in remote service

Reply:
RemoteService is a trait and not a struct which complicates your suggestion significantly.

Reply:
Overloaded terminology :)